docs/40_initialization.md

```diff
@@ -0,0 +1,446 @@
+<br/><br/>
+
+Next, find the `API Key` section and generate a new key.
+
+<br/><br/>
+<img
+src="../img/heroku_key.png"
+alt="heroku_app_name"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Keep this value for further steps.
+
+## AWS
+
+##### Create your organization's AWS account
+
+No special steps here, just create!
+
+
+##### Create an IAM User Group with `AdministratorAccess`
+
+<br/><br/>
+<img
+src="../img/user_group.png"
+alt="user_group"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Ensure that you click the right policy in the `Attach permissions policies` section.
+
+<br/><br/>
+<img
+src="../img/permissions.png"
+alt="permissions"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+**Note: If you know more refined permissions to give the IAM Group, that is preferred, as those with access
+to the credentials of `AdministratorAccess` in AWS can be an extreme security hazard if not managed appropriately**
+
+<br/><br/>
+<img
+src="../img/create_group.png"
+alt="create_group"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+
+##### Create an IAM user that lives in the User Group you made
+
+<br/><br/>
+<img
+src="../img/create_user.png"
+alt="create_user"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Next, give a unique name -- no need to enable Console Access
+
+<br/><br/>
+<img
+src="../img/specify_user.png"
+alt="specify_user"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Lastly, add them to the Group you made in the step above:
+
+<br/><br/>
+<img
+src="../img/add_perms.png"
+alt="add_perms"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Create security credentials
+
+You'll need to create `Security Credentials` for your User -- these credentials will be specifically used
+in your `Terraform Cloud` setup
+
+Firstly, go to your User and click the `Security Credentials` tab:
+
+<br/><br/>
+<img
+src="../img/creds.png"
+alt="creds"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+
+Navigate to `Create access key`
+
+<br/><br/>
+<img
+src="../img/access_key.png"
+alt="access_key"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+You'll be prompted to provide a reason for the access key creation. It doesn't matter much, so `Other` is a completely acceptable choice
+
+<br/><br/>
+<img
+src="../img/creds_purpose.png"
+alt="creds_purpose"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Time to create the key!
+
+<br/><br/>
+<img
+src="../img/create_tag.png"
```

Comment:
that is a lot of pngs... all adds up - would make git repo heavy etc.  
ATM  it is not that much -- almost 5MB
```
❯ git diff master... --stat | grep '\.png' | awk '{print $1;}' | xargs du -sck | grep total
4484	total
```
but if converted to .jpg and on such large ones sacrificing some quality -- we can get them 3 times smaller and only with minimal amount of distortions IMHO tollerable for this tech doc:
```
❯ git diff master... --stat | grep '\.png' | awk '{print $1;}' | while read f; do convert $f -quality 30 ${f//.png/.jpg}; done
❯ git diff master... --stat | grep '\.png' | sed -e 's,.png,.jpg,g' | awk '{print $1;}' | xargs eog
❯ git diff master... --stat | grep '\.png' | sed -e 's,.png,.jpg,g' | awk '{print $1;}' | xargs du -sck | grep total
1484	total
```
WDYT?

Reply:
Thanks Yarik.  Good catch.